src/AVFoundation/Enums.cs

```diff
@@ -643,7 +643,9 @@ public enum AVCaptureSessionInterruptionReason : nint {
 		VideoDeviceNotAvailableInBackground = 1,
 		AudioDeviceInUseByAnotherClient = 2,
 		VideoDeviceInUseByAnotherClient = 3,
-		VideoDeviceNotAvailableWithMultipleForegroundApps = 4
+		VideoDeviceNotAvailableWithMultipleForegroundApps = 4,
+		[iOS (11, 1)]
```

Comment:
NoMac, NoTV and NoWatch missing

Reply:
It already has a [NoTV, NoWatch] in the top of the enum and it is inside a #if !MONOMAC which means that is not necessary, right?

Reply:
Yeah technically it is not needed but I think it was the plan that at some point we would get rid of those conditionals if possible so having that metadata there won't hurt I guess, unless @spouliot says otherwise :)

Reply:
We'd get rid of `#if !MONOMAC`, not of availability attributes on the parent (so lack of `[NoTV, NoWatch]` still valid).

We could just add a `[NoMac]` to `public enum AVCaptureSessionInterruptionReason` but that would be just noise in this case, better do it for things we add.

Reply:
k, we'll soon have xtro results so it will be easier to spot them :)

Reply:
Ok, leaving as it is to reduce noise in the diff. We can do this later for new types or in a pr just to get rid of the #if !MONOMAC